bareos.job.py

```diff
@@ -34,14 +34,14 @@ def get_total_jobs(args):
     console = create_console()
     total_jobs = console.call(".sql query=\"SELECT COUNT(Job) FROM Job;\"")
     try:
-        print total_jobs['query'][0]['count(job)']
```

Comment:
Key was changed to 'count(job)' because of issue #1. Did you changed it back to 'count' intentionally?

Reply:
Yes this change was intentionally, with 'count(job)' it raises an exception and prints 0. 
I testes this only on version latest version 20.0.1 and it works fine

Reply:
Looks like in some cases it works with `count` key and in other cases it works with `count(job)` key. I will implement change which will check for both keys.